qemu/tests/kdump_with_stress.py

```diff
@@ -37,21 +38,23 @@ def install_stress_app(session):
         install_cmd = params.get("install_cmd")
 
         logging.info("Fetch package: '%s'" % link)
-        pkg = utils.unmap_url_cache(test.tmpdir, link, md5sum)
-        vm.copy_files_to(pkg, tmp_dir)
+        pkg_name = os.path.basename(link)
+        pkg_path = os.path.join(test.tmpdir, pkg_name)
```

Comment:
I think `test.tmpdir` is ok here.

Reply:
It will raise `IOError` if `dst` is a directory.
```python
>>> download.get_file('http://people.seas.harvard.edu/~apw/stress/stress-1.0.4.tar.gz', '/tmp', hash_expected='890a4236dd1656792f3ef9a190cf99ef')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/staf-kvm-py2/workspace/avocado/avocado/utils/download.py", line 165, in get_file
    hash_file = _verify_hash(_get_file(src, dst, permissions))
  File "/home/staf-kvm-py2/workspace/avocado/avocado/utils/download.py", line 120, in _get_file
    url_download(src, dst)
  File "/home/staf-kvm-py2/workspace/avocado/avocado/utils/download.py", line 70, in url_download
    with open(filename, 'wb') as dest_file:
IOError: [Errno 21] Is a directory: '/tmp'
```